src/Projection/InMemoryEventStoreQuery.php

```diff
@@ -396,7 +414,7 @@ private function prepareStreamPositions(): void
         }
 
         // stream names given
-        foreach ($this->query['streams'] as $stream) {
+        foreach ($this->query['streams'] as $key => $stream) {
```

Comment:
`$key` is unused and should be ignored, shouldn't it?

Reply:
yes, your right.